sandbox/errorHandling.lua

```diff
@@ -14,12 +17,24 @@ errorHandling = {}
 -- the latter is better in my eyes
 -- I am planning personally on waiting ECS changes have been implemented and I'm more sure on whats happening
 -- PlayerShips created via the gm screen are ***NOT*** wrapped - this would be fairly easy to fix via wrapping on onNewPlayerShip
--- If there is a use case people want this for I will add that happily
+
+function errorHandling:onError(err, replacementTraceback)
+	if self._onError then
+		if replacementTraceback then
+			self._onError(err,replacementTraceback)
+		else
+			-- it might be nice to suppress the first few lines of the traceback (the ones inside of errorHandling)

```

Comment:
That depends. I'll form an opinion after seeing a couple of errors with traceback lines